hvplot/tests/testinteractive.py

```diff
@@ -194,7 +195,7 @@ def sel_col(sel):
 
 
 def test_interactive_nested_widgets():
-    df = pd._testing.makeDataFrame()
+    df = makeDataFrame()
```

Comment:
Maybe let's start using the fixture more.

Reply:
The `df` fixture in this module that retunes `makeMixedDataFrame` is used a lot. Planning to refactor the tests in a future PR.